### PR TITLE
add kent instructor landing page

### PR DESCRIPTION
**Pull request by zacjones93**
Updated Kents page to include his products, podcast, case study, and a featured collection of content around getting into open source.

- [x] update instructor bio 

hype

<details>
<summary>screenshots</summary>

<br>

Learn web development from Kent C  Dodds on egghead  egghead io
Learn web development from Kent C  Dodds on egghead  egghead io

</details>

### CODE REVIEW

File: src/components/search/instructors/kent-c-dodds/index.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+           title,
+           description,
+           path,
+           byline,
+           image,
+           'background': images[label == 'feature-card-background'][0].url,
+           'instructor': collaborators[]->[role == 'instructor'][0]{
+             'name': person->.name
+           }
+         },
```
Comment (zacjones93) [inline src/components/search/instructors/kent-c-dodds/index.tsx:136]: https://www.loom.com/share/368bb9b44b59425abab8791a4f683afd

File: src/components/search/instructors/kent-c-dodds/index.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+            <VerticalResourceCard
+              resource={secondCourse}
+              className="w-1/2 border-none flex flex-col items-center justify-center text-center sm:py-8 py-6"
+              location="Kent C. Dodds instructor Landing page"
+            />
+            <VerticalResourceCard
+              className="w-1/2  border-none flex flex-col items-center justify-center text-center sm:py-8 py-6"
+              resource={caseStudy}
+              location="Kent C. Dodds instructor Landing page"
+            />
```
Comment (zacjones93) [inline src/components/search/instructors/kent-c-dodds/index.tsx:85]: The card components that we have built work really well

File: src/components/search/components/cta-card.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
+import Image from 'next/image'
+import Link from 'next/link'
+import {get} from 'lodash'
+
+import {bpMinMD} from 'utils/breakpoints'
+import {track} from 'utils/analytics'
+import ExternalTrackedLink from 'components/external-tracked-link'
+
+const CtaCard: React.FC<{location: string; trackTitle: string; resource: any}> =
```
Comment (zacjones93) [inline src/components/search/components/cta-card.tsx:9]: created a CTA Card in `/search/components` as this is a component that we are using consistently when updating instructor pages

**Comment by joelhooks** [comment]
should this be closed for inactivity?

**Comment by laurosilvacom** [comment]
@zacjones93 I think we move this forward with a small iterative step and get this merge by: 

1. Providing a full description in the bio section to reduce the white space. 
2. Replacing everything below the feature products with 2 rows of 3 or 4 course cards.
  <details>
  <summary>screenshot</summary>
  <br>
CleanShot 2021-08-09 at 23 51 22@2x
  </details>

3. Adding a bigger breakpoint to the product feature cards. 
  <details>
  <summary>screenshot</summary>
  <br>
  CleanShot 2021-08-09 at 23 53 54@2x
  </details>

**Comment by zacjones93** [comment]
Updated landing page with suggestions:

Learn web development from Kent C  Dodds on egghead  egghead io
Learn web development from Kent C  Dodds on egghead  egghead io

**Review by laurosilvacom — APPROVED** [review]
Looks good! We can iterate on this pattern as we curate additional instructor pages.

**Pull request merged**